resources/lang/id/layout.php

```diff
@@ -0,0 +1,263 @@
+            'beatmap_discussions-index' => 'laman diskusi beatmap',
+            'beatmapset-watches-index' => 'daftar pengamatan modding',
+            'beatmapset_discussion_votes-index' => 'laman diskusi voting beatmap',
+            'beatmapset_events-index' => 'laman peristiwa beatmap',
+            'index' => 'daftar',
+            'packs' => 'paket',
+            'show' => 'info',
+        ],
+        'beatmapsets' => [
+            '_' => 'beatmaps',
+            'discussion' => 'modding',
+        ],
+        'rankings' => [
+            '_' => 'peringkat',
+            'index' => 'performa',
+            'performance' => 'performa',
+            'charts' => 'spotlights',
+            'score' => 'skor',
+            'country' => 'negara',
+            'kudosu' => 'kudosu',
+        ],
+        'community' => [
+            '_' => 'komunitas',
+            'dev' => 'osu!dev',
+            'getForum' => 'forum',
+            'getChat' => 'obrolan',
+            'getLive' => 'siaran langsung',
+            'contests' => 'kontes',
+            'profile' => 'profil',
+            'tournaments' => 'turnamen',
+            'tournaments-index' => 'turnamen',
+            'tournaments-show' => 'info turnamen',
+            'forum-topic-watches-index' => 'langganan',
+            'forum-topics-create' => 'forum',
+            'forum-topics-show' => 'forum',
+            'forum-forums-index' => 'forum',
+            'forum-forums-show' => 'forum',
+        ],
+        'multiplayer' => [
+            '_' => 'multiplayer',
+            'show' => 'pertandingan',
+        ],
+        'error' => [
+            '_' => 'kesalahan',
+            '404' => 'hilang',
+            '403' => 'dilarang',
+            '401' => 'tidak memiliki akses',
+            '405' => 'hilang',
+            '500' => 'ada yang rusak',
+            '503' => 'pemeliharaan',
+        ],
+        'user' => [
+            '_' => 'pengguna',
+            'getLogin' => 'masuk',
+            'disabled' => 'nonaktif',
+
+            'register' => 'daftar',
+            'reset' => 'pulihkan',
+            'new' => 'baru',
+
+            'messages' => 'Pesan',
+            'settings' => 'Pengaturan',
+            'logout' => 'Keluar',
+            'help' => 'bantuan',
+            'modding-history-discussions' => 'diskusi modding pengguna',
+            'modding-history-events' => 'peristiwa modding pengguna',
+            'modding-history-index' => 'riwayat modding pengguna',
+            'modding-history-posts' => 'postingan modding pengguna',
+            'modding-history-votesGiven' => 'suara modding yang diberikan pengguna',
+            'modding-history-votesReceived' => 'suara modding yang diterima pengguna',
+        ],
+        'store' => [
+            '_' => 'toko',
+            'checkout-show' => 'periksa',
+            'getListing' => 'daftar',
+            'cart-show' => 'keranjang',
+
+            'getCheckout' => 'periksa',
+            'getInvoice' => 'fakture',
```

Comment:
Correct translation is actually "faktur" http://kbbi.web.id/faktur

+            'getInvoice' => 'faktur',